setup-stage2.sh

```diff
@@ -45,28 +47,11 @@ wine64 wineboot -fs &>/dev/null
 
 PROTON_VERSION_FULL=""
 
-if [[ -f "$PROTON_DIST_PATH/version" ]]; then
-    PROTON_VERSION_FULL="$(cat "$PROTON_DIST_PATH/version" | cut -d' ' -f2 | cut -d'-' -f1)"
-fi
-
-if [[ "$PROTON_VERSION_FULL" == "" ]]; then
-    PROTON_VERSION_FULL="$(echo "$PROTON_DIST_PATH" | grep -Po '\d\.\d+')"
-fi
-
-PROTON_VERSION_MAJOR="$(echo "$PROTON_VERSION_FULL" | cut -d'.' -f1)"
-PROTON_VERSION_MINOR="$(echo "$PROTON_VERSION_FULL" | cut -d'.' -f2)"
-
-if [[ "$PROTON_VERSION_FULL" == "" || "$PROTON_VERSION_MAJOR" == "" || "$PROTON_VERSION_MINOR" == "" ]]; then
-    error "Could not detect Proton version. Please request help in the #ffxiv-linux-discussion channel of the discord."
-    exit 1
-fi
-
 echo
-warn 'Note that the next step is destructive, meaning that if something goes wrong it can break your wine prefix and/or your proton runner installation.'
+warn 'Note that the next step is destructive, meaning that if something goes wrong it can break your wine prefix and/or your wine runner installation.'
 echo 'Please make backups of both!'
 echo "Wine prefix: $WINEPREFIX"
-echo "Proton distribution: $PROTON_DIST_PATH"
-echo "Proton version: ${PROTON_VERSION_MAJOR}.${PROTON_VERSION_MINOR}"
+echo "wine distribution: $PROTON_DIST_PATH"
```

Comment:
If the change is made to autodetect the wine version, it probably makes sense to print the detected version out here as well.